agasc/tests/test_agasc.py

```diff
@@ -37,9 +37,15 @@
     assert os.path.exists(ascrc_file)
     ascds_env = Ska.Shell.getenv('source {} -r release'.format(ascrc_file), shell='tcsh')
     assert 'ASCDS_BIN' in ascds_env
+    # Make a mp_get_agasc cmd line that just returns that command's status
+    cmd = 'mp_get_agasc -r 10 -d 20 -w 0.01 >& /dev/null; echo $?'
+    lines = Ska.Shell.tcsh(cmd, env=ascds_env)
+    # Check command status is 0
+    assert lines[0] == "0"
```

Comment:
Response in another window suggested I'd be getting non-zero status
```
ascds-quango: mp_get_agasc -r 10 -d 20 -w 0.01
/home/ascds/DS.release/bin/mp_get_agasc: line 81: /home/ascds/DS.release/bin/mp_get_agasc_exe: cannot execute binary file
ascds-quango: echo $?
126
```
but when I tried to set a breakpoint in run_shell to check, I didn't seem to get into the block at https://github.com/sot/Ska.Shell/blob/25fa2476e548bad25c284c76b8ef9005e96575c3/Ska/Shell/shell.py#L164
and then I lost interest.